python/cugraph/cugraph/dask/link_prediction/jaccard.py

```diff
@@ -140,21 +161,148 @@ def jaccard(input_graph, vertex_pair=None, use_weight=False):
 
     do_expensive_check = False
 
-    if vertex_pair is not None:
-        result = [
-            client.submit(
-                _call_plc_jaccard,
-                Comms.get_session_id(),
-                input_graph._plc_graph[w],
-                vertex_pair[w][0],
-                use_weight,
-                do_expensive_check,
-                vertex_pair_col_name,
-                workers=[w],
-                allow_other_workers=False,
+    result = [
+        client.submit(
+            _call_plc_jaccard,
+            Comms.get_session_id(),
+            input_graph._plc_graph[w],
+            vertex_pair[w][0],
+            use_weight,
+            do_expensive_check,
+            vertex_pair_col_name,
+            workers=[w],
+            allow_other_workers=False,
+        )
+        for w in Comms.get_workers()
+    ]
+
+    wait(result)
+
+    cudf_result = [client.submit(convert_to_cudf, cp_arrays) for cp_arrays in result]
+
+    wait(cudf_result)
+
+    ddf = dask_cudf.from_delayed(cudf_result).persist()
+    wait(ddf)
+
+    # Wait until the inactive futures are released
+    wait([(r.release(), c_r.release()) for r, c_r in zip(result, cudf_result)])
+
+    if input_graph.renumbered:
+        ddf = input_graph.unrenumber(ddf, "first")
+        ddf = input_graph.unrenumber(ddf, "second")
+
+    return ddf
+
+
+def all_pairs_jaccard(
+    input_graph,
+    vertices: cudf.Series = None,
+    use_weight: bool = False,
+    topk: int = None,
+):
+    """
+    Compute the All Pairs Jaccard similarity between all pairs of vertices specified.
```

Comment:
The description of the algo is identical to `all_pairs_cosine`, but the word "cosine" is replaced with "jaccard".  Shouldn't there be a different description?  Otherwise, it's not clear when I should use one over the other.

Reply:
I provided further description in the documentation